### PR TITLE
CR for Job #20254 - Additional Script updates

### PR DESCRIPTION
**Pull request by cozza13**
Code Review for Job #20254 - Workitem available at https://www.worklist.net/20254

### CODE REVIEW

File: examples/example/avatarcontrol/lookAtExample.js [hunk 1]
```diff
@@ -22,7 +22,6 @@ var oldMode = Camera.mode;
 function cancelLookAt() {
     if (lookingAtSomething) {
         lookingAtSomething = false;
-        Camera.stopLooking();
```
Comment (ZappoMan) [inline examples/example/avatarcontrol/lookAtExample.js]: why are you changing this?

File: examples/example/entities/editEntityExample.js [hunk 2]
```diff
@@ -74,7 +79,7 @@ function moveEntity(deltaTime) {
                 y: originalProperties.position.y + (count * positionDelta.y),
                 z: originalProperties.position.z + (count * positionDelta.z)
         },
-        radius : 0.25,
+        //radius : 0.05,
```
Comment (AndrewMeadows) [inline examples/example/entities/editEntityExample.js]: Remove the radius line completely rather than comment it out.

**Comment by cozza13** [comment]
@AndrewMeadows I have made your suggestion. 
@ZappoMan based on our conversation I made the changes as suggested.

Let me know before you hit merge so we can time it with some s3 work I want to do

File: examples/example/avatarcontrol/lookAtExample.js [hunk 3]
```diff
@@ -73,8 +73,8 @@ function mousePressEvent(event) {
             // switch to independent mode
             Camera.mode = "independent";
 
-            // tell the camera to fix it's look at on the point we clicked
-            Camera.keepLookingAt(intersection.intersection);
+              // tell the camera to fix it's look at on the point we clicked
+-            //Camera.keepLookingAt(intersection.intersection);
```
Comment (ZappoMan) [inline examples/example/avatarcontrol/lookAtExample.js]: you appear to have added a "-" to the front of this line... that will not end well. :)

**Pull request merged**